### PR TITLE
ROX-17332: Update docs to point to support matrix

### PR DESCRIPTION
**Pull request by kcarmichael08**
Version(s):
- 4.2+

[Issue](https://issues.redhat.com/browse/ROX-17332)

[Link to docs preview
](https://69757--docspreview.netlify.app/openshift-acs/latest/installing/acs-high-level-overview)

QE review:
- [ ] QE has approved this change.

Additional information:

- Removed support information and pointed users to the support documentation
- Consolidated information into "installation overview" module
- removed unused module/assembly

### CODE REVIEW

**Comment by mkumku** [comment]
Thank you. LGTM.

File: installing/installing_other/install-rhacs-other.adoc [hunk 1]
```diff
@@ -11,7 +11,7 @@ toc::[]
 
 Before you install:
 
-* Understand the xref:../../installing/acs-installation-platforms.adoc#install-platforms-methods_acs-installation-platforms[installation platforms and methods].
+* Understand the xref:../../installing/acs-high-level-overview.adoc#install-platforms-methods[installation methods for different platforms].
```
Comment (stevsmit) [inline installing/installing_other/install-rhacs-other.adoc:14]: I could not find this xref to ensure that it works!

File: installing/acs-high-level-overview.adoc [hunk 2]
```diff
@@ -1,23 +1,79 @@
...
-* xref:../installing/installing_ocp/install-central-ocp.adoc#install-central-ocp[Installing Central services for {product-title-short} on {osp}]
+[id="install-platforms-methods_{context}"]
+== Installation methods for different platforms
+
+[role="_abstract"]
+You can perform different types of installations on different platforms.
+
+[NOTE]
+====
+Not all installation methods are supported for all platforms. See the link:https://access.redhat.com/articles/7045053[Red Hat Advanced Cluster Security for Kubernetes Support Matrix].
```
Comment (stevsmit) [inline installing/acs-high-level-overview.adoc:33]: Kind of feels incomplete as is. Maybe not, though

```suggestion
Not all installation methods are supported for all platforms. See the link:https://access.redhat.com/articles/7045053[Red Hat Advanced Cluster Security for Kubernetes Support Matrix] for more information.
```

File: modules/installation-methods-for-different-architectures.adoc [hunk 3]
```diff
@@ -6,25 +6,18 @@
...
 
 |AMD64
-|Yes
-|Yes
 a|Operator (preferred), Helm charts, or `roxctl` CLI (not recommended)
 
 | ppc64le ({ibm-power})
-|Yes
-|Yes ({ocp} version 4.12 and later)
 .2+a|Operator (the only supported install method)
```
Comment (stevsmit) [inline modules/installation-methods-for-different-architectures.adoc:20]: Beyond scope of this but:

```suggestion
.2+a|Operator (the only supported installation method)
```

Alternatively, I wonder if you could just list "Operator" and leave out the "the only supported installation method" part.

**Comment by stevsmit** [comment]
@kcarmichael08 Looks really good! Just a few comments.

File: installing/acs-high-level-overview.adoc [hunk 4]
```diff
@@ -1,23 +1,79 @@
...
 
 toc::[]
 
 [role="_abstract"]
-{product-title} ({product-title-short}) provides security services for your self-managed {osp} Kubernetes systems or platforms such as Amazon Elastic Kubernetes Service (Amazon EKS), Google Kubernetes Engine (Google GKE), and Microsoft Azure Kubernetes Service (Microsoft AKS).
+{product-title} ({product-title-short}) provides security services for your self-managed {osp} Kubernetes systems or platforms such as {ocp}, Amazon Elastic Kubernetes Service (Amazon EKS), Google Kubernetes Engine (Google GKE), and Microsoft Azure Kubernetes Service (Microsoft AKS).
+
+For information on supported platforms and architecture, see the link:https://access.redhat.com/articles/7045053[Red Hat Advanced Cluster Security for Kubernetes Support Matrix]. For life cycle support information for {product-title-short}, see the link:https://access.redhat.com/support/policy/updates/rhacs[Red Hat Advanced Cluster Security for Kubernetes Support Policy].
+
+[discrete]
```
[the remark this answers is not in the document]
Comment (kcarmichael08) [reply installing/acs-high-level-overview.adoc:14]: Changed to non-discrete

File: installing/installing_other/install-rhacs-other.adoc (hunk 1 above)
Comment (kcarmichael08) [reply installing/installing_other/install-rhacs-other.adoc:14]: Thanks for catching that, somehow it wasn't included in the topic map!

File: modules/installation-methods-for-different-architectures.adoc [hunk 5]
```diff
@@ -6,25 +6,18 @@
 = Installation methods for different architectures
 
 [role="_abstract"]
-{rh-rhacs-first} supports the following architectures.
+{rh-rhacs-first} supports the following architectures. For information on supported platforms and architecture, see the link:https://access.redhat.com/articles/7045053[Red Hat Advanced Cluster Security for Kubernetes Support Matrix].
```
Comment (msugakov) [inline modules/installation-methods-for-different-architectures.adoc:9]: I'd suggest to rephrase this somewhat. E.g.

```suggestion
For information on supported platforms and architecture, see the link:https://access.redhat.com/articles/7045053[Red Hat Advanced Cluster Security for Kubernetes Support Matrix]. Additionally, the following table gives information about installation methods available per architecture.
```

File: modules/installation-methods-for-different-architectures.adoc [hunk 6]
```diff
@@ -6,25 +6,18 @@
...
 
-.Supported architectures and recommended installation methods
+.Architectures and recommended installation methods
```
Comment (msugakov) [inline modules/installation-methods-for-different-architectures.adoc:11]: [nit] For ppc64le and s390x there is only one _available_ option, it's not a _recommendation_, at least as it seems to me.

```suggestion
.Supported installation methods per architecture
```

**Review by msugakov — APPROVED** [review]
Makes sense and looks good as far as I can see.
Have two minor comments.

**Pull request merged**